sdcm/nemesis.py

```diff
@@ -2971,6 +3030,12 @@ def disrupt(self):
         time.sleep(300)
 
 
+class AddRemoveDcNemesis(Nemesis):
+
```

Comment:
We're going to be more specific with Nemesis properties, so please add:
disruptive = False
kubernetes = False
run_with_gemini = False
limited = True

Reply:
added